be/src/olap/rowset/segment_v2/segment.cpp

```diff
@@ -408,6 +408,7 @@ Status Segment::_create_column_readers(const SegmentFooterPB& footer) {
         RETURN_IF_ERROR(ColumnReader::create(opts, footer.columns(iter->second), footer.num_rows(),
                                              _file_reader, &reader));
         _column_readers.emplace(column.unique_id(), std::move(reader));
+        _meta_mem_usage += config::estimated_mem_per_column_reader;
```

Comment:
in past, `_meta_mem_usage` only tracking some indexes memory,
now, add column reader memory

will there be other memory in segment?

Reply:
It still has some unique_ptr/shared_ptr object and the object `SubcolumnColumnReaders`.